orttraining/orttraining/training_api/optimizer.cc

```diff
@@ -17,23 +17,14 @@ namespace api {
 
 namespace {
 
-// Currently all parameters are in a single group, so we hardcode group0 here.
 constexpr char GROUP_ZERO_NAME[] = "group0";
+static std::string FullQualifiedName_AdamWOptimizer(std::string(kMSDomain) + ":AdamWOptimizer");
+static std::string FullQualifiedName_SGDOptimizerV2(std::string(kMSDomain) + ":SGDOptimizerV2");
 
-// TODO: don't hard code the state names, should get the state names according to the optimizer types.
-// TODO: Consolidate with frontend tooling
-const std::vector<std::string> MOMENT_STATE_NAMES{"momentum0", "momentum1"};
-
-constexpr std::array AdamWOptimizerInputs = {
-    "learning_rate",
-    "step",
-    "params",
-    "gradients",
-    "first_order_moments",
-    "second_order_moments"};
+static const std::vector<std::string> CommonOptimizerInputs{"learning_rate", "step", "params", "gradients"};
```

Comment:
optional: this can be constexpr std::array since the value and size is known at compile time.